lambda/controller/create-replica/index.js

```diff
@@ -24,26 +24,42 @@ exports.handler = function(event, context, callback){
       return callback(err);
     }
 
+    var billingMode = 'PROVISIONED';
+
+    // Check the billing mode of the table
+    if (data.Table.hasOwnProperty('BillingModeSummary')){
+      if (data.Table.BillingModeSummary.BillingMode == 'PAY_PER_REQUEST') {
+        // console.log('PAY_PER_REQUEST');
+        billingMode = 'PAY_PER_REQUEST';
+      }
+    }
     //Construct replica table using source table description
     var params = {
       TableName: data.Table.TableName,
       AttributeDefinitions: data.Table.AttributeDefinitions,
       KeySchema: data.Table.KeySchema,
-      ProvisionedThroughput: {
-        ReadCapacityUnits: data.Table.ProvisionedThroughput.ReadCapacityUnits,
-        WriteCapacityUnits: data.Table.ProvisionedThroughput.WriteCapacityUnits
-      },
       StreamSpecification: {
         StreamEnabled: true,
         StreamViewType: 'NEW_AND_OLD_IMAGES'
       }
     };
 
+    // if the billingMode is PROVISIONED, need to add the throughput numbers
+    if (billingMode == 'PROVISIONED') {
```

Comment:
Awesome. Thanks for doing this.

Reply:
Chandra! Glad to have you looking at this - did I miss anything? Maybe a semicolon here or there?  :)